src/app/main/main.component.ts

```diff
@@ -124,9 +128,11 @@ export class MainComponent implements OnInit, OnDestroy, AfterViewInit  {
     this.geoColumnNames = [];
     this.rows = [];
 
-    this.project = this._route.snapshot.paramMap.get("project")
-    this.dataset = this._route.snapshot.paramMap.get("dataset")
-    this.table = this._route.snapshot.paramMap.get("table")
+    this.project = this._route.snapshot.paramMap.get("project");
+    this.dataset = this._route.snapshot.paramMap.get("dataset");
+    this.table = this._route.snapshot.paramMap.get("table");
+    this.jobID = this._route.snapshot.paramMap.get("job");
+    this.location = this._route.snapshot.paramMap.get("location");
```

Comment:
minor: Since these parameters are all used during initialization and then never again, maybe we should get them from the URL in `onSigninChange()` rather than making instance properties? Mainly to keep the list of instance properties a bit shorter.